p11-kit/uri.h

```diff
@@ -130,16 +130,16 @@ int                 p11_kit_uri_set_attributes              (P11KitUri *uri,
 
 void                p11_kit_uri_clear_attributes            (P11KitUri *uri);
 
-int                 p11_kit_uri_match_attributes            (P11KitUri *uri,
-                                                             CK_ATTRIBUTE_PTR attrs,
+int                 p11_kit_uri_match_attributes            (const P11KitUri *uri,
+                                                             const CK_ATTRIBUTE *attrs,
                                                              CK_ULONG n_attrs);
 
-const char*         p11_kit_uri_get_pin_value              (P11KitUri *uri);
+const char*         p11_kit_uri_get_pin_value               (const P11KitUri *uri);
 
-void                p11_kit_uri_set_pin_value              (P11KitUri *uri,
+void                p11_kit_uri_set_pin_value               (P11KitUri *uri,
                                                              const char *pin);
 
-const char*         p11_kit_uri_get_pin_source              (P11KitUri *uri);
+const char*         p11_kit_uri_get_pin_source              (const P11KitUri *uri);
```

Comment:
Should we add `const` to other `const char *` getters, such as `p11_kit_uri_get_module_name()` and `p11_kit_uri_get_module_path()`?